Wrappers.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "Cat.h"
+#include "VendingMachine.h"
+#include "Computer.h"
+#include "CatCyberOverlord.h"
+#include "CatRentalMachine.h"
```

Comment:
Don't include what you don't use. You aren't using any of these types in this header file. You are using pointers only.
That means you can forward declare all of these types instead.
Include these types in the CPP file instead
- [ ] revise